src/styles/main.scss

```diff
@@ -1,3 +1,19 @@
+@import 'https://fonts.googleapis.com/css?family=Roboto:100,100italic,300,300italic,regular,italic,500,500italic,700,700italic,900,900italic';
+@import './utils/variables';
+@import './blocks/page';
+@import './blocks/header/header';
+@import './blocks/header/nav';
+@import './blocks/header/is-active';
+@import './blocks/main/content/card';
+@import './blocks/main/content/button';
+@import './blocks//main/content/stars';
+@import './blocks/main/catalog';
```

Comment:
Also about file structure - I don't think you should put `card`, `button`, `stars` into `main/content` folders. Just put them into `blocks` folder directly